src/proto_bcd/findings/messages.py

```diff
@@ -27,6 +27,9 @@
 _templates[
     FindingCategory.ENUM_VALUE_NAME_CHANGE
 ] = "Existing value `{oldsubject}` is renamed to `{subject}` in enum `{context}`."
+_templates[
+    FindingCategory.ENUM_VALUE_NUMBER_CHANGE
+] = "Existing value `{oldsubject}` is changed to `{subject}` in enum `{context}`."
```

Comment:
Since `oldsubject` and `subject` will now be numbers here, we might want to rephrase this message, otherwise it will look like `Existing value 1 is changed to 2 in enum TestEnum`, which may be not too clear.

Reply:
Good point, I changed the subject strings.